neo/Network/RPC/RpcServer.cs

```diff
@@ -127,6 +127,8 @@ private static JObject GetRelayResult(RelayResultReason reason)
                     throw new RpcException(-503, "The block cannot be validated.");
                 case RelayResultReason.Invalid:
                     throw new RpcException(-504, "Block or transaction validation failed.");
+                case RelayResultReason.PolicyFail:
+                    throw new RpcException(-505, "One of the Policy filters failed [tx_size_limits_fees or blocked_accounts].");
```

Comment:
Let's move forward! In a future we change automatic numbering if necessary.. for now -500, -501 is very clear, which is good. Perhaps it will get worse by putting arithmetic here.